src/olivePage.ts

```diff
@@ -100,8 +101,9 @@ export default class OlivePage {
         Sorting.enableAjaxSorting($("a[data-sort]"));
         Sorting.setSortHeaderClass($("th[data-sort]"));
         Form.enablecleanUpNumberField($("[data-val-number]"));
-        Modal.enableEnsureHeight($("[data-toggle=tab]"));        
-        Select.enableEnhance($("select"));
+        Modal.enableEnsureHeight($("[data-toggle=tab]"));     
+        MultiSelect.enableEnhance($("select[data-control='collapsible-checkboxes']")); 
+        Select.enableEnhance($("select:not([data-control='collapsible-checkboxes'])"));
```

Comment:
This one work with the `chosen` library and serve drop-down list and other selectable type